app/src/main/java/woowacourse/paint/PaletteAdapter.kt

```diff
@@ -0,0 +1,20 @@
+package woowacourse.paint
+
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+
+class PaletteAdapter(
+    private val backgroundColors: List<Int>,
+    private val onClick: (Int) -> Unit,
```

Comment:
코틀린은 함수 타입을 선언할 때 매개 변수의 이름을 지정할 수 있습니다.
```suggestion
    private val onClick: (hexCode: Int) -> Unit,
```
위처럼 메서드의 의도를 나타낼 수 있게 매개 변수의 이름을 지정하면 어떨까요? 안드로이드 스튜디오에서 매개 변수 정보를 조회하면 매개 변수 이름까지 알 수 있기 때문에 `onClick` 메서드를 사용할 때 더 편하다고 생각합니다!

Reply:
오 위처럼 표현을 할 수 있었군요!👍